golang-example/README.md

```diff
@@ -16,10 +16,10 @@ To work with Go repositories you need to use [JFrog CLI](https://www.jfrog.com/c
 
 ```console
 Configure Artifactory:
-> jf c add
+> jf c add --url=<JFROG_PLATFORM_URL> [credentials flags]
```

Comment:
In my opinion, It is more confusing that way. It will still be an interactive command with the particular data set. I suggest adding a second non-interactive example.